(notes): PD 8.7.0 release notes (#20192)
OpenTrons/opentrons-api: protocol-designer/release-notes.md

---

## Opentrons Protocol Designer Changes in 8.7.0

**Welcome to Protocol Designer 8.7.0!**

> This release adds support for manual tip selection and camera steps in Protocol Designer, and includes feature improvements and bug fixes.

Reviewer: hmm there is only 1 camera step and its very basic right now

<img width="372" height="902" alt="Image" src="https://github.com/user-attachments/assets/190b59fa-345f-4120-b0c3-a347bb33d53e" />

```suggestion
This release adds support for manual tip selection and camera in Protocol Designer, and includes feature improvements and bug fixes.
```

Reviewer: @mjhuff @rclarke0 is camera only for Flex or both robots?

Reviewer 2: Both!

Reviewer 3: I like the original text. I interpret "camera steps" in the sense of "you can add one camera step or multiple camera steps to your protocol" — they all do the same thing, but you can have as many as you like.

Author: Yep; Ed's explanation was my intent. Just like we would say "you can add move steps to your protocol" even though there is only one form to open.

### New Features

> - Choose between automatic and manual tip tracking for transfer and mix steps in your protocols. In manual tip tracking, Protocol Designer lets you select the tip rack and individual tips the pipette will pick up to aspirate, dispense, or mix.

Reviewer 4: Nit, not sure we need to delineate between `aspirtate` and `dispense`. Maybe this can read `transfer or mix`?

- Manual tip tracking in Protocol Designer enables reusing tips more than once in a protocol.
- Add a camera step in any Protocol Designer protocol. The Flex or OT-2's built-in camera can take a still image of your robot deck at any point during the protocol. Access your images in the Recent Protocol Runs section of the Opentrons App's robot details page.

### Improved Features

- Protocol Designer only shows compatible deck and labware locations to move a lid to.
- Add Opentrons Tough Universal Lids to an available Opentrons Flex Deck Riser placed on the deck.
- Move a tip rack lid to any tip rack on the deck without a lid.

> - When adding labware, click tok add a tip rack lid to a Flex 96-channel tip rack adapter with a tip rack inside.

Reviewer: ```suggestion
- When adding labware, click too add a tip rack lid to a Flex 96-channel tip rack adapter with a tip rack inside.
```

Reviewer 3: `to` 😆

- When you click **Duplicate labware**, Protocol Designer duplicates all labware, adapters, lids, and liquids in the deck slot.

### Bug Fixes

> - Protocol Designer updates flow rate maximums update to the latest version when you import an older protocol.

Reviewer 3: 🤔 
```suggestion
- Protocol Designer updates maximum flow rates to the latest version when you import an older protocol.
```
